packages/components/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "rollup --config",
     "clean": "rm -rf dist/* tsconfig.tsbuildinfo",
     "prepublishOnly": "ts-node --project ../../tsconfig.bin.json scripts/entryPoints.ts",
-    "prepare": "npm run clean; npm run build",
+    "bootstrap": "npm run clean; npm run build",
```

Comment:
Reading this: https://lerna.js.org/docs/getting-started#package-dependency-management
It looks like the `@jobber` packages should be "*" is there a reason that wasn't changed?

Reply:
No reason other than it also worked and still symlinked so I didn't touch it haha

Reply:
Do you think we should update it to match what lerna expects?

Reply:
most likely yeah

Reply:
It works! 
![image](https://user-images.githubusercontent.com/15986172/227522060-569aa2e3-006e-4528-937c-dd739f7c39bf.png)